docs/user-guide/line-plot.ipynb

```diff
@@ -248,6 +248,92 @@
     "        linestyle={'a': 'solid', 'b': 'dashed'},\n",
     "        marker=None)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e485478c-3652-44fc-8b47-f368dad095cb",
+   "metadata": {},
+   "source": [
+    "## Bin edges\n",
+    "\n",
+    "When the coordinate of the data contains bin-edges, Plopp represents it as a step function instead of markers:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "32317d3d-2f05-417b-9004-8b0b047ab94a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da = pp.data.data_array(binedges=True)\n",
```

Comment:
I don't think this is very didactic. Can you just have a named function in `plopp.data`? Otherwise you unintentionally teach people the syntax (here: the `binedges` keyword arg) of a function that is irrelevant to them. How about `pp.data.histogram_1d()`? Maybe even something with "sample" or "example" in the name?

Reply:
I'm adding `data1d()`, `data2d()`, `histogram1d()`, `histogram2d()`, `dataset1d()`, `dataset2d()`